src/App.js

```diff
@@ -51,30 +51,103 @@ function App() {
   }, [todoList, isLoading]);
 
 
-  const addTodo = newTodo => {
+  const addTodo = async(newTodo) => {
     // setTodoList ([...todoList, newTodo]) USE PREVTODO!!!
-    setTodoList(prevTodoList => ([ ...prevTodoList, newTodo ]))
+   // setTodoList(prevTodoList => ([ ...prevTodoList, newTodo ]))
+   await postTodo(newTodo);
+  
   }  
+  const postTodo = async (newTodo) => {
+    try {
+      const airtableData = {
+        fields: {
+          title: newTodo.title
+        },
+      };
+  
+      const response = await fetch(
+        `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/`,
+        {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+            Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
+          },
+          body: JSON.stringify(airtableData),
+        }
+      );
   
-  const removeTodo = id => {
-    // setTodoList(prevTodoList.filter((todo) => todo.id !== id));
-    setTodoList(prevTodoList => prevTodoList.filter(todo => todo.id !== id));
+      if (!response.ok) {
+        const message = `Error has ocurred:${response.status}`;
+        throw new Error(message);
+      }
+  
+      const dataResponse = await response.json();
+      return dataResponse;
+    } catch (error) {
+      console.log(error.message);
+      return null;
+    }
+  };
+
+  const removeTodo = async(id) => {
+    await deleteTodo(id)
+  };
```

Comment:
Nitpick:
This `removeTodo` is only being used once below. I'd recommend renaming `deleteTodo` to `removeTodo` instead, which will reduce your code.